### ISSUE TITLE
Music information does not show in macOS control center Music widget

### ISSUE THREAD

**Issue by ceilingtilefan** [jeffvli/sonixd#454]
Sonixd does not show music information in the macOS control center Music widget. Adding the app to Accessibility > Media & Apple Music as well as Accessibility does not work.

![image](https://github.com/jeffvli/sonixd/assets/104149053/2c7c9b98-9d85-4eb4-8f8d-76b6a6a64ddf)

macOS Sonoma (14.4.1)
Version 0.15.5
